src/clientmetricreport/ClientMetricReport.ts

```diff
@@ -126,6 +126,35 @@ export default class ClientMetricReport {
     return Number(metricReport.currentMetrics[metricName] * 1000);
   };
 
+  millisecondsPerSecond = (metricName?: string, ssrc?: number): number => {
```

Comment:
Can we name this to be clearer what it does? milliseconds per second is kinda confusing

Reply:
Thanks for the comment. Created https://github.com/aws/amazon-chime-sdk-js/pull/2443 to update function naming